### ISSUE TITLE
Gitter suggestion

### ISSUE THREAD

**Issue by ghost** [rethinkdb/rethinkdb#3855]
Hello,

Is it possible to add a official channel RethinkDb in Gitter ?

Thank you

**Comment by joaojeronimo** [comment]
Even better, there's #rethinkdb on freenode ;)

**Comment by GeoCloud** [comment]
Ok but many developers used Gitter....

**Comment by internalfx** [comment]
IRC is so old and clunky compared to Gitter...

:+1:

**Comment by danielmewes** [comment]
I'm not sure if adding another channel for chats would be a good idea at this time.

It would add another place for our team to be available at, and in the end both #rethinkdb on IRC as well as the possible Gitter channel might suffer from the division.
(we already use Slack for internal communication, + the IRC channel for public chats)

Once RethinkDB is larger we will definitely consider this though.

**Comment by mglukhovsky** [comment]
@danielmewes, just a small note, Gitter supports IRC integration: https://irc.gitter.im/

I agree that adding another channel might divide our attention.

**Comment by danielmewes** [comment]
I didn't realize they had IRC support. In that case I think it's worth a try.

**Comment by deontologician** [comment]
That seems like it's for connecting to gitter with an irc client, vs. actually replicating messages across channels etc. To get the benefit, we'd need to abandon the freenode channel and move to the gitter irc channel. I guess it would be marginally easier for people who have irc open already to be in both channels, but it'd still be splitting things up

**Comment by danielmewes** [comment]
> I guess it would be marginally easier for people who have irc open already to be in both channels, but it'd still be splitting things up

That's what I thought. I don't think we should abandon the freenode channel. For starters not everyone has a GitHub account. Having two IRC channels doesn't sound too bad to me (we also won't advertise the Gitter one as an IRC channel. That's just how we internally can access it).

**Comment by kofalt** [comment]
> I didn't realize they had IRC support.

Unfortunately, I have found their IRC bridge support to be profoundly lacking, bordering on unusable; I regularly get disconnected. A support ticket acknowledged the problem and added that they're working on it, have not heard a follow-up. Combined with a web client that is confusingly heavy on resources, nagging prompts, and very slow, and it's overall an aggravating experience. 

Gitter has some cool ideas (orgs + repos maps to channels, GH usernames map, markdown) that I'd like to use, but the implementation is at this time sorely lacking. A few days after trying out Gitter for a team of ~10 people, we moved to HipChat, a change that was unanimously appreciated.

IMHO freenode is a much more dependable, scalable infrastructure on which to host community discussion until and unless Gitter radically improves their clients and servers.
